frontend/src/cards/ui/LawEnforcementAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import { MetricId } from "../../data/config/MetricConfig"
+import { DemographicType } from "../../data/query/Breakdowns"
+import { MetricQueryResponse } from "../../data/query/MetricQuery"
+import { ALL, BLACK_NH } from "../../data/utils/Constants"
+import { Row } from "../../data/utils/DatasetTypes"
+import { Fips } from "../../data/utils/Fips"
+import HetNotice from "../../styles/HetComponents/HetNotice"
+
+interface LawEnforcementAlertProps {
+    queryResponse: MetricQueryResponse
+    fips: Fips
+    demographicType: DemographicType
+}
+
+function LawEnforcementAlert(props: LawEnforcementAlertProps) {
+    const dataAlls: Row[] = props.queryResponse.data.filter(
+        (row) => row[props.demographicType] === ALL
+    )
+
+    const dataBlack: Row[] = props.queryResponse.data.filter(
+        (row) => row[props.demographicType] === BLACK_NH
+    )
+
+    const totalCountId: MetricId = 'gun_violence_legal_intervention_estimated_total'
+
+    const totalCount: number | undefined = dataAlls.length > 0 ? dataAlls[0][totalCountId] : undefined
```

Comment:
could probably simplify this with optional chaining right? `dataAlls?.[0]?.[totalCountId] or something like that